routes/users.js

```diff
@@ -25,11 +72,11 @@ router.get('/', authenticate, usersController.getUsers)
  *             schema:
  *               $ref: '#/components/schemas/users'
  *       401:
- *         description: Unauthorized
+ *         description: unauthorized
  *         content:
  *           application/json:
  *             schema:
- *               $ref: '#/components/schemas/errors/unAuthorized'
+ *               $ref: '#/components/schemas/errors/unauthorized'
```

Comment:
NIT: Revert this line, keep the `unAuthorized` in camel case.

Reply:
okay sure!

Reply:
Done with changes!